<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="jenkins-agent-10-25-224-5" name="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" tests="6" failures="0" timestamp="2023-03-09T07:55:14 UTC" time="60.313" errors="0">
  <testcase name="invokeAsyncTestWith_502" time="13.054" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_429" time="5.059" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_429_And_No_MaxRetryTime" time="0.046" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_504" time="2.050" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_503" time="40.052" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_504_And_No_MaxRetryTime" time="0.052" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
</testsuite> <!-- com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest -->
